core/domain/acl_decorators.py

```diff
@@ -343,24 +343,89 @@ def test_can_access(self, **kwargs):
     return test_can_access
 
 
-def can_comment_on_feedback_thread(handler):
-    """Decorator to check whether the user can view feedback for a given
-    exploration.
+def can_create_feedback_thread(handler):
+    """Decorator to check whether the user can create a feedback thread.
     """
 
     def test_can_access(self, exploration_id, **kwargs):
-        """Checks if the user can view the exploration feedback.
+        """Checks if the user can create a feedback thread.
 
         Args:
-            exploration_id: str. The exploration id.
+            exploration_id: str. The ID of the exploration where the thread will
+                be created.
+            **kwargs: *. Keyword arguments.
+
+        Returns:
+            bool. Whether the user can create the feedback thread.
```

Comment:
I don't see a boolean being returned anywhere. I think I made a comment on this to @apb7 recently too...

Reply:
I noticed that too. But the rest of the file had 'bool. description' so I followed it. I'm not sure what type would be appropriate to describe 'handler(self, exploration_id, **kwargs)'. Probably 'function. The function to be called if the user can create the feedback thread' sounds better?

Reply:
It's returning the _result_ of the handler call, not the function, right? (please check)

If so, then probably something like `*. The return value of the decorated method.` -- perhaps?

Reply:
Oh correct. My bad. I'll edit it appropriately. Thanks!

Reply:
Done

Reply:
@seanlip: I have added `acl_decorators` to Issue #4374. Also, @nithusha21 did you correct the return for this instance or for others as well? Thanks!

Reply:
@seanlip: I have added `acl_decorators` to Issue #4374. Also, @nithusha21 did you correct the return for this instance or for others as well? Thanks!

Reply:
@apb7 -- I corrected return only for the decorators I added. Did not edit the other instances.